opentelemetry-api/pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-api"
+description = "OpenTelemetry Python API"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.6"
+authors = [
+    { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Typing :: Typed",
+]
+dependencies = [
+    "aiocontextvars; python_version<'3.7'",
+    "Deprecated >= 1.2.6",
+    "setuptools >= 16.0",
+]
+dynamic = [
+    "version",
+]
+
+[project.optional-dependencies]
+test = []
+
+[project.entry-points.opentelemetry_context]
+contextvars_context = "opentelemetry.context.contextvars_context:ContextVarsRuntimeContext"
+
+[project.entry-points.opentelemetry_environment_variables]
+api = "opentelemetry.environment_variables"
+
+[project.entry-points.opentelemetry_meter_provider]
+default_meter_provider = "opentelemetry.metrics:NoOpMeterProvider"
+
+[project.entry-points.opentelemetry_propagator]
+baggage = "opentelemetry.baggage.propagation:W3CBaggagePropagator"
+tracecontext = "opentelemetry.trace.propagation.tracecontext:TraceContextTextMapPropagator"
+
+[project.entry-points.opentelemetry_tracer_provider]
+default_tracer_provider = "opentelemetry.trace:NoOpTracerProvider"
+
+[project.urls]
+Homepage = "https://github.com/open-telemetry/opentelemetry-python/tree/main/opentelemetry-api"
+
+[tool.hatch.version]
+path = "src/opentelemetry/version.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
```

Comment:
I'm not familiar with how `hatchling` works. Do these need to be explicitly included unlike [sdist](https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute? Why is `README.rst` not included?

Reply:
Some files are always included https://hatch.pypa.io/latest/plugins/builder/sdist/#default-file-selection